src/services/car/messageLead.ts

```diff
@@ -55,3 +62,31 @@ export const sendMessageLead = async ({
     })
   }
 }
+
+export const fetchDealerMessageLeads = async ({
+  dealerId,
+  page = 0,
+  size = 7,
+  options = {},
+}: {
+  dealerId: number
+  page: number
+  size?: number
+  options?: ApiCallOptions & { validateOnly?: boolean }
+}): Promise<Paginated<MessageLeadsListItem>> => {
+  const { validateOnly, ...otherOptions } = options
+
+  const path = createApiPathWithValidate(
+    `dealers/${dealerId}/message-leads`,
+    validateOnly
+  )
```

Comment:
No need for `validateOnly` since were fetching objects and not creating new ones